src/isolate_create_params.rs

```diff
@@ -192,6 +192,7 @@ pub(crate) mod raw {
     pub embedder_wrapper_type_index: int,
     pub embedder_wrapper_object_index: int,
     pub cpp_heap_params: SharedPtr<CppHeapCreateParams>,
+    // NOTE(bartlomieju): this field is deprecated in V8 API.
```

Comment:
Use deprecated attribute? https://doc.rust-lang.org/edition-guide/rust-2018/the-compiler/an-attribute-for-deprecation.html

Reply:
I don't think that's a good idea, it will show up as a warning and this field cannot be used/configured by user whatsoever.